projects/court-case-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/api/proxy/CommunityApiController.kt

```diff
@@ -57,7 +59,68 @@ class CommunityApiController(
             log.error("Exception thrown when calling ${communityApiUrl + request.requestURI}. community-api returned ${ex.message}")
             ResponseEntity.status(ex.statusCode)
                 .headers(ex.responseHeaders)
-                .body(ex.responseBodyAsString);
+                .body(ex.responseBodyAsString)
+        }
+    }
+
+    @PostMapping("/compare")
+    fun compare(@RequestBody compare: Compare, request: HttpServletRequest): CompareReport {
+        val headers = request.headerNames.asSequence().associateWith { request.getHeader(it) }
+            .filter { it.key.lowercase() == "Authorization".lowercase() }.toMutableMap()
```

Comment:
Is this just getting the auth header?  Could it be simplified to something like:

```suggestion
        val headers = mapOf("Authorization" to request.getHeader("Authorization"))
```

Reply:
Fixed

Reply:
fixed